src/test/bctest.py

```diff
@@ -38,7 +38,7 @@ def bctest(testDir, testObj, exeext):
 	if "return_code" in testObj:
 		wantRC = testObj['return_code']
 	if proc.returncode != wantRC:
-		print("Return code mismatch for " + outputFn)
+		print("Return code mismatch for " + execprog + ' ' + str(execargs))
```

Comment:
While testing, I discovered that when we hit this error case, python stack-traces (unhandled exception) because `outputFn` is not defined. This change makes it much clearer which test case failed. (This seems to be still broken in bitcoin.)